process/sync/baseSync.go

```diff
@@ -111,15 +111,16 @@ func (boot *baseBootstrap) loadBlocks(
 		}
 
 		if err == nil {
+			err = boot.accounts.RecreateTrie(boot.blkc.GetCurrentBlockHeader().GetRootHash())
```

Comment:
In the current version, the error is always nil. Does this change anything in the logic that calls this method?

Reply:
ok